pkg/cmd/clientConfig.go

```diff
@@ -70,18 +78,41 @@ kubectl plugin mobile get clientconfig`,
 				}
 				ret = append(ret, svcConfig)
 			}
-
 			outputJSON := ServiceConfigs{
 				Services:  ret,
 				Namespace: ns,
 			}
-
-			encoder := json.NewEncoder(os.Stdout)
-			encoder.SetIndent("", "  ")
-			if err := encoder.Encode(outputJSON); err != nil {
-				return err
+			if clientID, _ := cmd.Flags().GetString("client-id"); clientID != "" {
+				outputJSON.ClientID = clientID
+			}
+			if err := ccc.Out.Render("get"+cmd.Name(), outputType(cmd.Flags()), outputJSON); err != nil {
+				return errors.Wrap(err, fmt.Sprintf(output.FailedToOutPutInFormat, "ServiceConfig", outputType(cmd.Flags())))
 			}
-			return err
+			return nil
 		},
 	}
+
+	ccc.Out.AddRenderer("get"+cmd.Name(), "table", func(writer io.Writer, serviceConfigs interface{}) error {
+		scL := serviceConfigs.(ServiceConfigs)
+		var data [][]string
```

Comment:
could you please rename `scL` it is hard to know what it is without looking it up.